### ISSUE TITLE
template php-fpm config

### ISSUE THREAD

**Issue by bendavies** [usabilla/php-docker-template#119]
Hi there,

Would you be open to a PR to allow templating php-fpm config via env vars?

Something like this, as these are the values that would mostly likely need scaling/configuring:
```
pm = ${PHP_FPM_PM}
pm.max_children = ${PHP_FPM_PM_MAX_CHILDREN}
pm.start_servers = ${PHP_FPM_PM_START_SERVERS}
pm.min_spare_servers = ${PHP_FPM_PM_MIN_SPARE_SERVERS}
pm.max_spare_servers = ${PHP_FPM_PM_MAX_SPARE_SERVERS}
pm.process_idle_timeout = ${PHP_FPM_PM_PROCESS_IDLE_TIMEOUT}
pm.max_requests = ${PHP_FPM_PM_MAX_REQUESTS}
```

Thanks.

**Comment by renatomefi** [comment]
Hello, so sorry for the delay!
Yes we'd like to have that, we were actually thinking about doing it as well! Is it still something you're willing to do?

**Comment by bendavies** [comment]
@renatomefi yeah, i can probably!
Would you be interested in only php fpm, or php.ini settings too?

**Comment by renatomefi** [comment]
@bendavies for now I'd say let's do the fpm config!
You can do something similar to how we do with nginx config:
- https://github.com/usabilla/php-docker-template/blob/master/src/http/nginx/docker-nginx-entrypoint#L5
- https://github.com/usabilla/php-docker-template/blob/master/src/http/nginx/conf/main/nginx.conf.template

Also, if you're comfortable with it you can write some pytests to validate it, but if not I can help you out after you're done with the functionality!

Thanks again

**Comment by bendavies** [comment]
@renatomefi PR opened!

**Comment by rdohms** [comment]
Closed by #125 and #126
